ui_espace_pro/src/pages/CfaCandidatInformationPage/CfaCandidatInformationPage.jsx

```diff
@@ -0,0 +1,129 @@
+    }
+  }, [utmSource])
+
+  return (
+    <FormLayoutComponent
+      headerText={
+        <>
+          Un candidat souhaite <br />
+          être contacté <br />
+          <Text textStyle="h2" as="span" color="grey.700">
+            par votre centre de formation
+          </Text>
+        </>
+      }
+    >
+      {loading && <span>Chargement des données...</span>}
+      {data?.user && (
+        <Box mt={10}>
+          <Text as="span" color="bluefrance" textStyle="h6">
+            Voici les coordonnées du contact
+          </Text>
+          {data.user?.type && (
+            <Text fontWeight="700" mb={4}>
+              {data.user.type === "parent" ? "Le parent" : "L'étudiant"}
+            </Text>
+          )}
+          <Box mt={6} pb={8} borderBottom="solid 1px #D0C9C4">
+            <Text>
+              Nom :{" "}
+              <Text as="span">
+                <strong>{data.user.lastname}</strong>
+              </Text>
+            </Text>
+            <Text>
+              Prénom :{" "}
+              <Text as="span">
+                <b>{data.user.firstname}</b>
+              </Text>
+            </Text>
+            <Text mt={4}>
+              Numéro de téléphone :{" "}
+              <Text as="span">
+                <b>{data.user.phone.match(/.{1,2}/g).join(".")}</b>
+              </Text>
+            </Text>
+            <Text>
+              Email :{" "}
+              <Text as="span">
+                <b>{data.user.email}</b>
+              </Text>
+            </Text>
+          </Box>
+          <Box mt={8} pb={8} borderBottom="solid 1px #D0C9C4">
+            <Text as="p" my="2">
+              Il ou elle souhaite aborder avec vous le(s) sujet(s) suivant :
+            </Text>
+            <Text as="p" bg="#F6F6F6" color="#2A2A2A" fontSize="16px" lineHeight="24px" fontWeight="700" px="4" py="2">
+              {data.appointment.applicant_message_to_cfa}
+            </Text>
+            <Text as="p" mt="2">
+              à propos de la formation : <strong>{data.etablissement.training_intitule_long}</strong>
+            </Text>
+            <Text as="p" mt="1">
+              dispensée par :{" "}
+              <strong>
+                {data.etablissement.etablissement_formateur_raison_sociale}, {data.etablissement.lieu_formation_street}, {data.etablissement.lieu_formation_zip_code},{" "}
+                {data.etablissement.lieu_formation_city}
+              </strong>
+            </Text>
+          </Box>
+          {currentState === "initial" ? <CfaCandidatInformationForm formik={formik} setCurrentState={setCurrentState} /> : <></>}
+          {currentState === "sending" ? <div>Envoi en cours...</div> : <></>}
+          {currentState === "answered" ? <CfaCandidatInformationAnswered msg={formik.values.message} /> : <></>}
+          {currentState === "other" ? <CfaCandidatInformationOther /> : <></>}
+          {currentState === "unreachable" ? <CfaCandidatInformationUnreachable /> : <></>}
```

Comment:
`currentState` could be constant values instead of hard coded values

Reply:
Yes but it would be somehow over-engineering to define constants/enums for such a simple use case. Moreover, hard-coded string are a lot more flexible (if the state is shared across multiple subcomponents).